scripts/action-recognition/train_recognizer.py

```diff
@@ -180,6 +180,15 @@ def batch_fn(batch, ctx):
                                                    new_width=opt.new_width, new_height=opt.new_height, new_length=opt.new_length,
                                                    target_width=input_size, target_height=input_size,
                                                    num_segments=opt.num_segments, transform=transform_test)
+    elif opt.dataset == 'somethingsomethingv2':
+        train_dataset = somethingsomethingv2.classification.SomethingSomethingV2(setting=opt.train_list, root=data_dir, train=True,
```

Comment:
line width and indent seems weird, can we import `SomethingSomethingV2` directly?

Reply:
Sure, I will change them all.